crates/bevy_ecs/src/system/commands/entity_command.rs

```diff
@@ -323,12 +324,17 @@ pub fn move_components<B: Bundle>(target: Entity) -> impl EntityCommand {
 /// An [`EntityCommand`] that logs the components of an entity.
 pub fn log_components() -> impl EntityCommand {
     move |entity: EntityWorldMut| {
+        let name = entity
+            .get::<Name>()
+            .map(|name| format!(" ({name})"))
+            .unwrap_or_default();
+        let id = entity.id();
         let debug_infos: Vec<_> = entity
             .world()
             .inspect_entity(entity.id())
             .expect("Entity existence is verified before an EntityCommand is executed")
-            .map(ComponentInfo::name)
+            .map(|info| info.name().to_string())
```

Comment:
just leaving a note here that this isn't as_string because as_string only exists on `debug` whereas `to_string` handles the debug/no-debug difference